cmake/OpenCVFindOpenEXR.cmake

```diff
@@ -94,17 +94,20 @@ FOREACH(SEARCH_PATH ${SEARCH_PATHS})
             IF (${contents} MATCHES "#define OPENEXR_VERSION_MINOR ([0-9]+)")
                 SET(OPENEXR_VERSION_MINOR "${CMAKE_MATCH_1}")
             ENDIF ()
+	    FILE (STRINGS ${OPENEXR_VERSION_FILE} contents REGEX "#define OPENEXR_VERSION_PATCH ")
+	    IF (${contents} MATCHES "#define OPENEXR_VERSION_PATCH ([0-9]+)")
+                SET(OPENEXR_VERSION_PATCH "${CMAKE_MATCH_1}")
```

Comment:
There is something wrong with code indentation.

Reply:
Thank you for your comment ! To fix it, I replaced tab to space.